CafeTime/CafeTime/AuthManager.swift

```diff
@@ -61,22 +112,23 @@
     // MARK: User
     
     func userCredentials() -> CurrentUserCredentialsRealm? {
-        return AuthDAO().localUserCredentials
+        return DAO().getLocalUserCredentials()
     }
     
-    
     func logOutUser() {
         if let auth = FIRAuth.auth() {
             do {
                 try auth.signOut()
-                let dao = AuthDAO()
+                let dao = DAO()
```

Comment:
probably better to have some name for DAO, cause in future you can add more of them, and just to separate functionality 
DAO is a very common name. AuthDAO - gives an idea of what it does
it's fine for now, cause you don't have others